docker/dev_vpp_agent/README.md

```diff
@@ -212,9 +212,8 @@ The ETCD server will be available on your host OS IP (most likely
 Call the agent via ETCD using the testing client:
 ```
 cd $GOPATH/src/github.com/ligato/vpp-agent/cmd/vpp-agent-ctl
-go run main.go /opt/vpp-agent/dev/etcd.conf -ct
-go run main.go /opt/vpp-agent/dev/etcd.conf -mt
-go run main.go /opt/vpp-agent/dev/etcd.conf -dt
+go run main.go /opt/vpp-agent/dev/etcd.conf -tap
```

Comment:
Forgot to change `main.go` here. I would suggest to use same way here as elsewhere, simply `vpp-agent-ctl /opt/vpp-agent/dev/etcd.conf -tap`.